proposals/lip-0042.md

```diff
@@ -206,4 +210,5 @@ TBA
 
```

Comment:
In the `afterTransactionsExecute` you should call `getBlockReward` with the block assets as well. Keep in mind the format that you end up choosing for the assets (decoded / not decoded).

Reply:
Added, for the moment I am considering it as encoded.